wfdb/processing/peaks.py

```diff
@@ -4,7 +4,7 @@
 
 
 def find_peaks(sig):
-    """
+    r"""
```

Comment:
typo?

Reply:
No, this has to be a raw string because it contains the sequence `\/`, which otherwise would be interpreted as an invalid escape sequence.